docs/en/observability/apm-ui/troubleshooting.asciidoc

```diff
@@ -42,6 +42,8 @@ This section can help with any of the following:
 * You see a message like "No Services Found",
 * You see errors like "Fielddata is disabled on text fields by default..."
 
+// Assuming the APM integration is no longer responsible for index templates and ingest pipelines,
+// what is the fix now?
 These problems are likely to be caused by a missing index template or ingest pipeline.
 By default, {fleet} sets up these and other APM assets when the APM integration is installed.
 Try reinstalling the APM integration by navigating to
```

Comment:
In the troubleshooting guide we tell users that if no data is appearing in the APM UI, the cause could be missing index mappings and the fix is to reinstall the APM integration. Assuming the APM integration is no longer responsible for index mappings, what is the fix now? Or will this issue not happen anymore?